specs.bzl

```diff
@@ -4,26 +4,27 @@
 #
 
 def _maven_repository(url, user = None, password = None):
-    """
-    Generates the data map for a Maven repository specifier given the available information. If both a user
-    and password are given as arguments, it will include the access credentials in the repository spec. If one or
-    both are missing, it will just generate the repository url.
+    """Generates the data map for a Maven repository specifier given the available information.
+
+    If both a user and password are given as arguments, it will include the
+    access credentials in the repository spec. If one or both are missing, it
+    will just generate the repository url.
 
     Args:
-        url: *Required* A string containing the repository url (ex: `"https://maven.google.com/"`)
-        user: *Optional* A username for this Maven repository, if it requires authentication (ex: `"bob"`)
-        password: *Optional* A password for this Maven repository, if it requires authentication (ex: `"l0bl4w"`)
-
-    Output Schema:
-        {
-            "repo_url": String
-            "credentials: Optional Map
-                {
-                    "username": String
-                    "password": String
-                }
-        }
-    """
+        url: A string containing the repository url (ex: "https://maven.google.com/").
+        user: A username for this Maven repository, if it requires authentication (ex: "johndoe").
+        password: A password for this Maven repository, if it requires authentication (ex: "example-password").
+    """
+
+    # Output Schema:
```

Comment:
Why is this now commented out? Here and below

Reply:
I wasn't sure if this output schema is useful to end users to be included in Stardoc. Also, `Args:` needs to be in the final section of the macro documentation.

Reply:
Oh I didn't notice it was outside of the docs section. Yeah, good call. I think it's fine not in the official docs